apps/ledger-live-desktop/src/renderer/components/RecoverBanner/RecoverBanner.tsx

```diff
@@ -29,7 +29,7 @@ export default function RecoverBanner({ children }: { children?: React.ReactNode
   const recoverServices = useFeature("protectServicesDesktop");
 
   const recoverBannerIsEnabled = recoverServices?.params?.bannerSubscriptionNotification;
-  const protectID = recoverServices?.params?.protectId ?? "";
+  const protectID = recoverServices?.params?.protectId ?? "protect-prod";
```

Comment:
We should probably have a constant for `protect-prod` as I see it repeated multiple times

Reply:
Agreed 👍 , maybe a subject for the recover team though